gcm_toolkit/utils/interface.py

```diff
@@ -225,6 +293,7 @@ def _set_data_common(self, time, tag=None, regrid_lowres=False):
 
         self.dsi = dsi
         self.chemistry.set_data(dsi)
+        self.chemistry.abunds = xr.Dataset()
```

Comment:
Here, I changed it so abunds get reset. This allows to reuse Interface class instances, otherwise you have the old abundances left in the chemistry module.